src/orion/core/io/experiment_builder.py

```diff
@@ -206,8 +206,9 @@ def build_view_from(self, cmdargs):
 
         # Information should be enough to infer experiment's name.
         log.debug("Creating %s database client with args: %s", dbtype, db_opts)
+
         try:
-            Database(of_type=dbtype, **db_opts)
```

Comment:
Why remove `of_type`? It should still work based on `Dataset.__init__`, right?

Reply:
Because the `Database` wrapper takes care of constructing the dictionary, it just removes unnecessary fluff from the call and reduces the complexity of the API, since you don't have to remember to call through explicit arguments